File: Lab5.py
for i in range(200,300) :
	Sum1 = 0
	for j in range(1,i):
	    if i % j == 0 :
		    Sum1 += j

	for k in range(200,300):
		Sum2 = 0
		for j in range(1,k):
			if k % j == 0:
				Sum2 += j

		if i == Sum2 and k == Sum1:
			print("Pair of friendly numbers:" + str(i) + " and " + str(k))

File: Lab5/Lab5.cpp
#include <iostream>
using namespace std;
int main()
{
	int Sum1;
	int Sum2;
	int i, k, j;
	for (i = 200; i < 300; i++) {
		Sum1 = 0;
		for (j = 1; j < i; j++) {
			if (i % j == 0) {
				Sum1 += j;
		}
		}
		for (k = 200; k < 300; k++) {
			Sum2 = 0;
			for (j = 1; j < i; j++) {
				if (k % j == 0) {
					Sum2 += j;
				}
			}
			if (i == Sum2 && k == Sum1) {
				cout << "Pair of friendly numbers:" << i << " and " << k;
			}
		}
	}
}
